peer/brontide.go

```diff
@@ -1245,6 +1249,53 @@ func (p *Brontide) maybeSendNodeAnn(channels []*channeldb.OpenChannel) {
 	}
 }
 
+// maybeSendChannelUpdates sends our channel updates to the remote peer if we
+// have any active channels with them.
+func (p *Brontide) maybeSendChannelUpdates(channels []*channeldb.OpenChannel) {
+	defer p.wg.Done()
+
+	// If we don't have any active channels, then we can exit early.
+	if len(channels) == 0 {
+		return
+	}
+
+	for _, dbChan := range channels {
+		switch {
+		// Skip anything that isn't the default channel state.
+		case !dbChan.HasChanStatus(channeldb.ChanStatusDefault):
+			continue
+
+		// If we need to do an SCB flow, a channel update is useless.
+		case !dbChan.HasChanStatus(channeldb.ChanStatusRestored):
```

Comment:
should remove `!` here?

Reply:
Good catch! I simplified to just filter if it's default for not, looking at the implementation default means none of the other status flags set: https://github.com/lightningnetwork/lnd/blob/c46b1a467afd96109c9e5f1fced1eae8aed6c98d/channeldb/migration25/channel.go#L323-L331